context/app/static/js/components/files/FileBrowser/style.js

```diff
@@ -1,9 +1,27 @@
 import styled from 'styled-components';
-import Paper from '@material-ui/core/Paper';
+import TableBody from '@material-ui/core/TableBody';
+import TableContainer from '@material-ui/core/TableContainer';
+import TableHead from '@material-ui/core/TableHead';
 
-const ScrollPaper = styled(Paper)`
-  max-height: 600px;
+const ChipWrapper = styled.div`
+  border-bottom: 1px solid ${(props) => props.theme.palette.divider};
+  padding: 12px 15px 12px 15px;
```

Comment:
```suggestion
  padding: 12px 15px;
```
Since the the pair match?